1-mandatory/2-people-i-know.js

```diff
@@ -378,14 +378,23 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+for (let i = 0; i < myFriends.length; i++) {
```

Comment:
If you look at the "Checks" tab on this PR page, you can see some of the tests didn't pass.
https://github.com/CodeYourFuture/JavaScript-Core-2-Coursework-Week2/pull/273/checks 

If you look at the logs it also tells you why.

The following error shows:
`ReferenceError: myFriends is not defined

      379 |
      380 | let thirtyFiveOrOlder = [];
    > 381 | for (let i = 0; i < myFriends.length; i++) {
          |                     ^
      382 |   if (myFriends[i].age >= 35) {
      383 |     thirtyFiveOrOlder.push(myFriends[i]);
      384 |   }

      at Object.<anonymous> (1-mandatory/2-people-i-know.js:381:[21](https://github.com/CodeYourFuture/JavaScript-Core-2-Coursework-Week2/pull/273/checks#step:4:22))
`

So it's pretty much explaining myFriends is not defined, it doesn't know what it is.

The reason for this is the variable array you want to loop through is actually called friends and not myFriends (see at the beginning of 2-people-i-know.js)